### ISSUE TITLE
paperless-ngx Redis Connection Fails

### ISSUE THREAD

**Issue by RikuXan** [gabe565/charts#551]
### Chart Name

paperless-ngx

### Chart Version

0.15.1

### Bug Description

My paperless-ngx cannot connect to Redis anymore, printing out an authentication error. Using the same Redis connection URL as in the paperless-ngx environment variable via redis-cli inside the Redis container gives me the same error. To connect successfully I have to either explicitly include the `default` username inside the connection string or alternatively remove the colon before the password.

**Comment by gabe565** [comment]
@RikuXan I suspect this is related to the recent Redis Helm chart update. I'm working on a fix!

**Comment by gabe565** [comment]
@RikuXan It's interesting because [their Redis env documentation](https://docs.paperless-ngx.com/configuration/#redis-broker) still shows the syntax including the `:`, so I'm hesitant to update the default value. I'm able to replicate, though. Hmm

**Comment by gabe565** [comment]
@RikuXan Oh whoops...I'm not actually able to replicate. It was a network policy issue in my setup. My pod seems to be healthy, and the env got set to `PAPERLESS_REDIS=redis://:KZ8fXPJJSC@paperless-ngx-redis-master`. Do you have any redis configuration set other than the default?

**Comment by RikuXan** [comment]
@gabe565 No, my Redis configuration is completely default. The only customizations I'm currently using in the overall chart are:

- Ingress and persistence configurations
- Using `existingSecrets`s for paperless & PostgreSQL credentials
- Pinning the PostgreSQL Image Version to a 15.x (haven't gotten around to do the manual major upgrade process)
- Extending the containers and environment variables with Tika and Gotenberg

My Redis connection string looks the same as yours, but using it via redis-cli (`redis-cli -u <connection-string>`) inside the Redis container gives me the aforementioned error. The Redis container's image version is `7.2.4-debian-11-r0`.

If it's not an issue for you, a solution might be to include a variable that is empty by default, but can be customized for the username. This way people who experience the issue could customize their Redis connection string, as well as creating compatibility with externally-hosted Redis instances with ACLs enabled (where the username can then actually be relevant as far as I understand it.
